vars/request.go

```diff
@@ -54,62 +55,27 @@ func (rp Request) Fill(holders []string) map[string]string {
 	return vars
 }
 
-func (rp Request) getScheme() (string, bool) {
-	if rp.Request.OriginalRequest.TLS != nil {
-		return "https", true
-	}
-
-	return "http", true
-}
-
-func (rp Request) getHostname(m *definition.Mock, req *definition.Request) (string, bool) {
-	if len(rp.Request.Host) == 0 {
-		return "", false
-	}
-	value := rp.Request.Host
-
-	return value, true
-}
-
-func (rp Request) getPort() (string, bool) {
-	host := rp.Request.OriginalRequest.Host
-	if len(host) == 0 {
-		return "", false
-	}
-
-	index := strings.Index(host, ":")
-	if index > -1 {
-		return host[index+1:], true
-	}
-
-	return "", false
-}
-
 func (rp Request) getUrlShort() (string, bool) {
-	host := strings.Split(rp.Request.OriginalRequest.Host, ":")[0]
-	if len(host) == 0 {
-		return "", false
+	if len(rp.Request.Port) == 0 || rp.Request.Port == "80" {
+		return fmt.Sprintf("%s://%s", rp.Request.Schema, rp.Request.Host), true
```

Comment:
Hmm... can't fully agree. My idea was to have the url short always without the port, so that it is scheme + hostname. Scheme + hostname + port would be returned by `getUrl()`.

But maybe I am thinking too our-usage specific, when we have the mmock running on the same hostname as the product, but using a different port. I wanted to have a single placeholder to get the product hostname with the scheme without having to use `{{request.scheme}}://{{request.host}}